src/lib/logMsg/logMsg.h

```diff
@@ -981,11 +981,14 @@ do                                                                       \
 /* ****************************************************************************
 *
 * LM_T - log trace message
+*
+* FIXME: temporal change, just for Orion contextBroker, use LogLevelDebug for LM_T
```

Comment:
You mean temporal in the context of logMsg library but permanent in the cotext of Oiron. Is that correct?